opensrs/opensrsapi.py

```diff
@@ -288,6 +288,12 @@ def _set_domain_contacts(self, cookie, user, domain):
                 'tech': contact,
             },
         }
+
+        if domain.lower().endswith('.ca'):
```

Comment:
I think we already switched to operate domain names in lower case.  So, it's probably no need to lower here.  (just don't like when we pollute our code with those `.lower()` in all possible places).